### PR TITLE
completed html-relative-links exercise. - Steve Behm 

### PR DESCRIPTION
**Pull request by SteveBehm**
Screen Shot 2022-01-25 at 11 00 16 AM
Screen Shot 2022-01-25 at 11 00 24 AM
Screen Shot 2022-01-25 at 11 00 33 AM
Screen Shot 2022-01-25 at 11 00 43 AM

### CODE REVIEW

File: html-relative-links/about-me/about-me.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+</head>
+<body>
+  <ul>
+    <li><a href="../index.html">Home Page</a></li>
+    <li><a href="about-me.html">About Me</a></li>
+    <li><a href="../projects/projects.html">Projects</a></li>
+    <li><a href="../contact-us/contact-us.html">Contact Us</a></li>
+  </ul>
+  <p>Hello About Me!</p>
+  <img src="https://www.mlive.com/resizer/HmVkldbVbQ08gk0FCJSZo-ZRVfY=/1280x0/smart/cloudfront-us-east-1.images.arcpublishing.com/advancelocal/HB6R5CPZ7RCGXN6MAHIAHLYHVY.jpg" alt="Detroit Red Wings Hockey Team">
```
Comment (codingcodymiller) [inline html-relative-links/about-me/about-me.html:19]: Please use a relative url to the images provided in the starter files.

File: html-relative-links/contact-us/contact-us.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+</head>
+<body>
+  <ul>
+    <li><a href="../index.html">Home Page</a></li>
+    <li><a href="../about-me/about-me.html">About Me</a></li>
+    <li><a href="../projects/projects.html">Projects</a></li>
+    <li><a href="contact-us.html">Contact Us</a></li>
+  </ul>
+  <p>Hello Contact Us!</p>
+  <img src="https://www.howtogeek.com/wp-content/uploads/2019/03/gmail-1.png?width=1198&trim=1,1&bg-color=000&pad=1,1" alt="Gmail Logo">
```
Comment (codingcodymiller) [inline html-relative-links/contact-us/contact-us.html:19]: Please use a relative url to the images provided in the starter files.

File: html-relative-links/index.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+</head>
+<body>
+  <ul>
+    <li><a href="index.html">Home Page</a></li>
+    <li><a href="about-me/about-me.html">About Me</a></li>
+    <li><a href="projects/projects.html">Projects</a></li>
+    <li><a href="contact-us/contact-us.html">Contact Us</a></li>
+  </ul>
+  <p>Hello Homepage!</p>
+  <img src="https://i0.heartyhosting.com/www.surfer.com/wp-content/uploads/2020/10/SR1020_PROSURF_u2329430.jpg?resize=1920%2C1080&ssl=1" alt="Guy surfing big wave">
```
Comment (codingcodymiller) [inline html-relative-links/index.html:19]: Please use a relative url to the images provided in the starter files.

File: html-relative-links/projects/projects.html [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+</head>
+<body>
+  <ul>
+    <li><a href="../index.html">Home Page</a></li>
+    <li><a href="../about-me/about-me.html">About Me</a></li>
+    <li><a href="projects.html">Projects</a></li>
+    <li><a href="../contact-us/contact-us.html">Contact Us</a></li>
+  </ul>
+  <p>Hello Projects!</p>
+  <img src="https://miro.medium.com/max/1400/1*v3IHHAYzA9olTKiNksMUCQ.png" alt="Example of Code">
```
Comment (codingcodymiller) [inline html-relative-links/projects/projects.html:19]: Please use a relative url to the images provided in the starter files.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: html-relative-links/about-me/about-me.html (hunk 1 above)
Comment (SteveBehm) [reply html-relative-links/about-me/about-me.html:19]: Screen Shot 2022-01-25 at 11 13 59 AM

File: html-relative-links/contact-us/contact-us.html (hunk 2 above)
Comment (SteveBehm) [reply html-relative-links/contact-us/contact-us.html:19]: Screen Shot 2022-01-25 at 11 13 40 AM

File: html-relative-links/index.html (hunk 3 above)
Comment (SteveBehm) [reply html-relative-links/index.html:19]: Screen Shot 2022-01-25 at 11 13 32 AM

File: html-relative-links/projects/projects.html (hunk 4 above)
Comment (SteveBehm) [reply html-relative-links/projects/projects.html:19]: Screen Shot 2022-01-25 at 11 13 49 AM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**